bundles/org.openhab.binding.zoneminder/src/main/resources/OH-INF/i18n/zoneminder.properties

```diff
@@ -111,3 +113,7 @@ channel-type.zoneminder.triggerAlarm.label = Trigger Alarm
 channel-type.zoneminder.triggerAlarm.description = Triggers an alarm
 channel-type.zoneminder.url.label = URL
 channel-type.zoneminder.url.description = URL of image or stream
+
+# thing types
+
+thing-type.zoneminder.server.channel.runState.label = Run State
```

Comment:
This can now be removed.

Reply:
Hmm. That's strange. I reran `mvn i18n:generate-default-translations`. Shouldn't that have removed it?

Reply:
It doesn't remove strings, it only moves to the bottom those that are not statically declared.